packages/web3-shared/base/src/specs/index.ts

```diff
@@ -1087,7 +1089,7 @@ export interface HubState<
 
 export interface IdentityServiceState {
     /** Find all social addresses related to the given identity. */
-    lookup(identity: SocialIdentity): Promise<Array<SocialAddress<NetworkPluginID>>>
+    lookup(identity: SocialIdentity, isOwnerIdentity?:boolean): Promise<Array<SocialAddress<NetworkPluginID>>>
```

Comment:
Read from the first parameter identity.

```suggestion
    lookup(identity: SocialIdentity): Promise<Array<SocialAddress<NetworkPluginID>>>
```